src/components/DataAvailabilityTable/header/CustomTblHeader.js

```
@@ -0,0 +1,92 @@
+                  />
+                );
+              }
+
+              return (
+                <HeaderCell
+                  components={components}
+                  column={column}
+                  sortBy={sortBy}
+                  sortOrder={sortOrder}
+                  toggleSort={() => sortByColumn(column.dataField, sortOrder)}
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)